### PR TITLE
Added Bootstrap project

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Xcode
```
Comment (pkondrashkov) [inline .gitignore]: Add section for MacOS and exlude .DS_Store please

File: Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//
+//  EmptyPresenterImpl.swift
+//  Bootstrap
+//
+//  Created by Alex Yanski on 3/5/20.
+//  Copyright © 2020 Alex Yanski. All rights reserved.
+//
+
+import Foundation
```
Comment (pkondrashkov) [inline Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift]: Remove unused import please

File: Bootstrap/Presentation/Empty/EmptyViewController.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+//
+//  Created by Alex Yanski on 3/3/20.
+//  Copyright © 2020 Alex Yanski. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class EmptyViewController: UIViewController {
+    var contentView: UIView = UIView()
```
Comment (pkondrashkov) [inline Bootstrap/Presentation/Empty/EmptyViewController.swift]: Why this is var and not private?

File: Bootstrap/Presentation/Root/RootContract.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+protocol RootBuilder: AnyObject {
+    func build() -> UIViewController
+}
+
+protocol RootRouter: AnyObject {
+    func showEmpty()
+}
+
+protocol RootViewModel {}
```
Comment (pkondrashkov) [inline Bootstrap/Presentation/Root/RootContract.swift]: There should be no such thing as ViewModel. Remove please :)

File: Podfile [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```
Comment (pkondrashkov) [inline Podfile]: Let's uncomment that and make sure the project targeted SDK is also 9.0

Comment (AlexYanski) [reply Podfile]: Fixed version to 10.0, because SnapKit supports 10.0 or higher

File: Bootstrap/Presentation/Root/RootContract.swift (hunk 4 above)
Comment (AlexYanski) [reply Bootstrap/Presentation/Root/RootContract.swift]: Done

File: Bootstrap/Presentation/Empty/EmptyViewController.swift (hunk 3 above)
Comment (AlexYanski) [reply Bootstrap/Presentation/Empty/EmptyViewController.swift]: Fixed

File: Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift (hunk 2 above)
Comment (AlexYanski) [reply Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift]: Done

File: .gitignore (hunk 1 above)
Comment (AlexYanski) [reply .gitignore]: Done

File: Podfile (hunk 5 above)
Comment (AlexYanski) [reply Podfile]: Change back to iOS 9.0

File: Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift [hunk 6]
```diff
@@ -0,0 +1,27 @@
+//
+//  EmptyPresenterImpl.swift
+//  Bootstrap
+//
+//  Created by Alex Yanski on 3/5/20.
+//  Copyright © 2020 Alex Yanski. All rights reserved.
+//
+
+final class EmptyPresenterImpl: EmptyPresenter {
+    private unowned let view: EmptyView
```
Comment (pkondrashkov) [inline Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift]: lets make it weak.

File: Bootstrap/Presentation/Root/RootComponent.swift [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+//
+//  Created by Alex Yanski on 3/4/20.
+//  Copyright © 2020 Alex Yanski. All rights reserved.
+//
+
+import UIKit
+
+final class RootComponent: EmptyDependency {
+    private let dependency: RootDependency
+    let parent: UINavigationController
```
Comment (pkondrashkov) [inline Bootstrap/Presentation/Root/RootComponent.swift]: unowned?

File: Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift (hunk 6 above)
Comment (AlexYanski) [reply Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift]: done

File: Bootstrap/Presentation/Root/RootComponent.swift (hunk 7 above)
Comment (AlexYanski) [reply Bootstrap/Presentation/Root/RootComponent.swift]: done

File: Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift (hunk 6 above)
Comment (pkondrashkov) [reply Bootstrap/Presentation/Empty/EmptyPresenterImpl.swift]: ? where is it done? still unowned

**Pull request closed without merging**